api/client/client.go

```diff
@@ -139,42 +139,35 @@ func (c *Client) NewCollection(ctx context.Context, dbID thread.ID, config db.Co
 }
 
 // Create creates new instances of objects
-func (c *Client) Create(ctx context.Context, dbID thread.ID, collectionName string, items ...interface{}) error {
-	values, err := marshalItems(items)
+func (c *Client) Create(ctx context.Context, dbID thread.ID, collectionName string, items ...interface{}) ([]string, error) {
```

Comment:
Client `Create` now returns list of instance ids, doesn't modify instances passed in.